src/styles/main.scss

```diff
@@ -10,5 +10,50 @@ body {
   box-sizing: border-box;
 }
 
+@import "./variables.scss";
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media (max-width: 487px) {
+
+  .catalog {
+    display: grid;
+    grid-template-columns: $cardWidth;
+    gap: 48px;
+    justify-content: center;
+    align-content: center;
+  }
+}
+
+@media (min-width: 488px) {
+
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(2, $cardWidth);
+    gap: 48px;
+    justify-content: center;
+    align-content: center;
+  }
+}
+
+@media (min-width: 768px) {
+
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(3, $cardWidth);
+    gap: 48px;
+    justify-content: center;
+    align-content: center;
+  }
+}
```

Comment:
You need add in media only grid-template-colums, other attributes must be allways in catalog, i think it's not serious mistake, but it will be better

Reply:
yes it would be better! Thanks